central/scannerdefinitions/handler/handler.go

```diff
@@ -494,19 +502,23 @@ func openFromArchive(archiveFile string, fileName string) (*os.File, error) {
 }
 
 func (h *httpHandler) getV4Files(w http.ResponseWriter, r *http.Request, updaterKey, fileName string) {
-	log.Infof("Fetching repository mapping file: %s", fileName)
+	log.Debugf("Fetching scanner V4 file: %s", fileName)
 
 	var err error
 	var f *vulDefFile
 
-	switch updaterKey {
-	case mappingUpdaterKey:
-		f, err = h.openMostRecentFile(mappingUpdaterKey, fileName)
-	default:
-		errMsg := fmt.Sprintf("internal error: invalid updater key: %s", updaterKey)
-		log.Error(errMsg)
-		httputil.WriteGRPCStyleErrorf(w, codes.Internal, errMsg)
-		return
+	if mappingUpdaterKey == updaterKey {
+		f, err = h.openMostRecentV4File(mappingUpdaterKey, fileName)
+	} else {
+		if strings.Contains(strings.ToLower(updaterKey), strings.ToLower("-nightly")) {
```

Comment:
May be able to use `version.GetVersionKind()` here to determine if its a nighly, dev version, etc.
https://github.com/stackrox/stackrox/blob/da365c3ccdcd5593cb9db5c85af443c261080df4/pkg/version/kind.go#L40